packages/cashier/src/components/cashier-locked/cashier-locked-provider.tsx

```diff
@@ -334,6 +336,18 @@ const getMessage = ({
     }
 
     if (is_deposit_locked) {
+        if (is_duplicate_dob_phone) {
```

Comment:
As there's no else, lets keep it in the same for easier readability? 
`if(is_deposit_locked && is_duplicate_dob_phone)`

Reply:
@habib-deriv Unfortunately, I can't do this, because there are some more conditions there (image attached). You can see there are 3 more conditions we have to check if `is_deposit_locked === true`. I've added `is_duplicate_dob_phone` at the top because it has the highest priority.

<img width="859" alt="image" src="https://github.com/user-attachments/assets/59fa0030-bd88-4c62-afe4-073633d1f36f">

Reply:
Ah noted. I guess it was formating issue with github showing. LGTM then